de-de/README.md

```diff
@@ -0,0 +1,39 @@
+# 🇩🇪 Deutsch (Deutschland)
+
+Willkommen beim Fabulously Optimized Wiki!
+
+[⬇️ Modpack installieren](de-de/einrichtung.md) | [🆕 Modpack aktualisieren](de-de/update-anleitung.md) [(Einstellungen zurücksetzen)](de-de/update-anleitung.md#einstellungen-zurücksetzen) | [➕ Mods hinzufügen](de-de/mods-hinzufuegen.md) | [➖ Mods deaktivieren](de-de/mods-deaktivieren.md) | [🎛️ Geänderte Optionen](de-de/geaenderte-optionen.md) [(behobene Fehler)](de-de/geaenderte-optionen.md#behobene-fehler) [(wie konfiguriere ich Mods?)](de-de/geaenderte-optionen.md#mods-konfigurieren)
+
+Kriege [🦸 einen kostenlosen Umhang](de-de/kostenloser-umhang.md) | [🌅 Shaders](de-de/shader-installieren.md) | [🖼️ Ressourcenpakete zum Laufen](de-de/ressourcenpaket-probleme.md)
```

Comment:
Capitalized letter here and in other links
```suggestion
Kriege [🦸 Einen kostenlosen Umhang](de-de/kostenloser-umhang.md) | [🌅 Shaders](de-de/shader-installieren.md) | [🖼️ Ressourcenpakete zum Laufen](de-de/ressourcenpaket-probleme.md)
```

Reply:
This isn't grammatically correct. `Kriege einen` not `Kriege Einen`.

Reply:
Ahh, yeah
I forgot the first word was there :D